### DISCUSSION TITLE
Should `/account/status` be there as a client API?

### DISCUSSION THREAD

**Discussion by typed-sigterm** [appwrite/appwrite#8765] [General]
`PATCH /account/status` is a client API that allow users to block themselves. I'm wondering about why this needs to be provided as a client API.

Blocking users is usually a privilege of admins. Just like `DELETE /users/{userId}`, this API will have the impact on the user's access to the account. Admins may see unexpected results when searching for blocked users. Although a **normal** user is impossible to F12+Postman to block themselves...

**Comment by DH-555** [comment]
Why would an user want to block himself and use postman, etc to block himself?

Anyways, I think this is a great feature/change request so I recommend creating an issue here in GitHub for this 😄
